src/setup.c

```diff
@@ -336,7 +336,7 @@ static void setup_adc2(void)
  * - Configure the base time (no clock division ratio, no aligned mode,
  *   direction up).
  * - Set clock division, prescaler and period parameters to get an update
- *   event with a frequency of 16 KHz. 16 interruptions by ms, 4 sensors with
+ *   event with a frequency of 1.6 KHz. 1.6 interruptions by ms, 4 sensors with
```

Comment:
The sensors readings problems have been already solved with commit 0325166e2d34e5306f4434fd457c3ef6d6062e98, so it is no needed to change the frequency.

Reply:
You are right! Done. :blush: